### DISCUSSION TITLE
Getting 403 for OpsgenieAddr

### DISCUSSION THREAD

**Discussion by sucremad** [jertel/elastalert2#952] [Q&A] [answered]
Hi All,

I am getting 403 Client Error while i trying to send alerts to opsgenie. I tried both addresses (standart, and eu) but still getting same error. 

`INFO:elastalert:Error response from https://api.eu.opsgenie.com/v2/alerts
 API Response: <Response [403]>
ERROR:elastalert:Error while running alert opsgenie: Error sending alert: 403 Client Error: Forbidden for url: https://api.eu.opsgenie.com/v2/alerts`

`
INFO:elastalert:Error response from https://api.opsgenie.com/v2/alerts
 API Response: <Response [403]>
ERROR:elastalert:Error while running alert opsgenie: Error sending alert: 403 Client Error: Forbidden for url: https://api.opsgenie.com/v2/alerts
`

What can I do else? What am i missing? could you please help?

Thank you.

**Comment by jertel** [comment, marked as answer]
From the OpsGenie documentation: 

```
All requests of Team Member API work under Team domain. It means that; If you are using apiKey belongs to a team integration, that team must allowed to access given team. Otherwise, we would respond with 403 - Forbidden (Unauthorized) with 40301 internal code (Response)
```
I suggest checking that the OpsGenie and API Key permissions are correct. The OpsGenie team may be in a better position to help you.
